.rubocop.yml

```diff
@@ -36,7 +36,7 @@ Metrics/MethodLength:
   # There's ~2 long methods in `StripeClient` and one in `NestedResource`. If
   # we want to truncate those a little, we could move this to be closer to ~30
   # (but the default of 10 is probably too short).
-  Max: 55
+  Max: 60
```

Comment:
This was necessary because of the additions to the [execute_request_internal](https://github.com/stripe/stripe-ruby/pull/1210/files#diff-fe0e4dffe7e252a01f235338ad30c7e0343bd0369a074fe40e0330784349db0dR433) method. Wasn't sure of a great way to split up that method into smaller chunks.

Reply:
That same method also caused a linting error ([build failure](https://github.com/stripe/stripe-ruby/actions/runs/4682575682/jobs/8296597342?pr=1210)), looking into that